web/src/integrationTest/kotlin/no/kartverket/matrikkel/bygning/v1/common/JWTUtil.kt

```diff
@@ -0,0 +1,32 @@
+package no.kartverket.matrikkel.bygning.v1.common
+
+import no.nav.security.mock.oauth2.MockOAuth2Server
+
+class JWTUtil() {
+    companion object {
+        private const val DEFAULT_SUBJECT = "123456789"
+        private const val DEFAULT_ISSUER = "testIssuer"
+        private const val DEFAULT_ORG = "123456789"
+
+        fun MockOAuth2Server.getJWTWithScope(scope: String) = issueToken(
+            issuerId = DEFAULT_ISSUER,
+            subject = DEFAULT_SUBJECT,
+            claims = mapOf(
+                "orgno" to DEFAULT_ORG,
+                "scope" to scope,
+            ),
+        )
+
+        fun MockOAuth2Server.getDefaultMaskinportenJWT() = getJWTWithScope("kartverk:riktig:scope")
```

Comment:
Foreslår å kalle dette `issue` for å tydeliggjøre at man genererer et token (og for å henge mer sammen med metodenavnene på mockOauth serveren).

```suggestion
        fun MockOAuth2Server.issueMaskinportenJWT() = getJWTWithScope("kartverk:riktig:scope")
```

Reply:
Kunne det også vært en idé at scope tas inn som et argument i `issueMaskinportenJWT` slik at man bare kaller samme funksjon, men med et ugyldig sope? Da kan antagelig `getJWTWithScope` fjernes?

```
//gyldig token
issueMaskinportenJWT()
//ugyldig scope
issueMaskinportenJWT("feil:scope")

// gyldig idporten token
issueIDPortenJWT()
```